Tests/Data/Parabolic/T/3D_3BHEs_array/bcs_tespy.py

```diff
@@ -0,0 +1,186 @@
+import sys
```

Comment:
license/copyright

Reply:
added in [dda3c74](https://github.com/ufz/ogs/pull/2702/commits/dda3c74aefbd9e1569ad8bf7cc55f166f0b6cac6)